content/go/concepts/strings/terms/cutprefix/cutprefix.md

```diff
@@ -0,0 +1,79 @@
+---
+Title: 'CutPrefix()'
+Description: 'Checks if a given prefix exists on the specified string. Returns the provided string without the prefix and true if found or the provided string and false if the string does not have the prefix.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`CutPrefix()`** function in Go is used to return a specified string without a given prefix string. The function returns the specified string without the given prefix and `true` if the given string has the specified prefix, otherwise it returns the full string and `false`. If the given prefix is an empty string, the function returns the full string and `true`.
```

Comment:
```suggestion
The **`CutPrefix()`** function in Go is used to return a specified string without a given prefix. The function returns the specified string without the given prefix and boolean value that confirms the presence of the prefix. It returns`true` if the given string has the specified prefix, otherwise it returns the full string and `false`. If the given prefix is an empty string, the function returns the full string and `true`.
```